di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/config/CriType.java

```diff
@@ -7,7 +7,8 @@ public enum CriType {
     FRAUD_CRI_TYPE("FRAUD"),
     VERIFICATION_CRI_TYPE("VERIFICATION"),
     USER_ASSERTED_CRI_TYPE("USER_ASSERTED"),
-    DOC_CHECK_APP_CRI_TYPE("DOC_CHECK_APP");
+    DOC_CHECK_APP_CRI_TYPE("DOC_CHECK_APP"),
+    CLAIMED_IDENTITY_CRI_TYPE("CLAIMED_IDENTITY");
```

Comment:
Wondering if it can just be `USER_ASSERTED` same as address as there's no scoring to do?